### PR TITLE
DDLC quotes

### PR DESCRIPTION
**Pull request by WoahItsKarma**
Added quotes from Doki Doki Literature Club!

### CODE REVIEW

**Review by Miodec — CHANGES_REQUESTED** [review]
Please replace all the new lines with spaces.

**Review by Miodec — DISMISSED** [review]

**Comment by Miodec** [comment]
The build failed, please verify the JSON structure

**Comment by Bruception** [comment]
Still failing

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 866,
+      "id": 5935
+    },
+    {
+      "text": "A marvel millions of years in the making. Where the womb of Earth chaotically meets the surface. Under a clear blue sky, an expanse of bliss - But beneath gray rolling clouds, an endless enigma. The easiest world to get lost in is one where everything can be found. One can only build a sand castle where the sand is wet. But where the sand is wet, the tide comes. Will it gently lick at your foundations until you give in? Or will a sudden wave send you crashing down in the blink of an eye? Either way the outcome is the same. Yet we still build sand castles. I stand where the foam wraps around my ankles. Where my toes squish into the sand. The salty air is therapeutic. The breeze is gentle, yet powerful. I sink my toes into the ultimate boundary line, tempted by the foamy tendrils. Turn back, and I abandon my peace to erode at the shore. Drift forward, and I return to Earth forevermore.",
+      "source": "Doki Doki Literature Club!",
+      "length": 898,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34993]:
```suggestion
      "length": 896,
```

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 1160,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34981]:
```suggestion
      "length": 1157,
```

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -34902,6 +34902,96 @@
...
+      "id": 5931
+    },
+    {
+      "text": "Your mind is so full of troubles and fears. That diminished your wonder over the years. But today I have a special place. A beach for us to go. A shore reaching beyond your sight. A sea that sparkles with brilliant light. The walls in your mind will melt away before the sunny glow. I'll be the beach that washes your worries away. I'll be the beach that you daydream about each day. I'll be the beach that makes your heart leap. In a way you thought had left you long ago. Let's bury your heavy thoughts in a pile of sand. Bathe in sunbeams and hold my hand. Wash your insecurities in the salty sea. And let me see you shine. Let's leave your memories in a footprint trail. Set you free in my windy sail. And remember the reasons you're wonderful. When you press your lips to mine. I'll be the beach that washes your worries away. I'll be the beach that you daydream about each day. I'll be the beach that makes your heart leap in a way you thought had left you long ago. But if you let me by your side. Your own beach, your own escape. You'll learn to love yourself again.",
+      "source": "Doki Doki Literature Club!",
+      "length": 1062,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34969]:
```suggestion
      "length": 1074,
```

File: frontend/static/quotes/english.json [hunk 4]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 958,
+      "id": 5930
+    },
+    {
+      "text": "Tomorrow will be brighter with me around. But when today is dim, I can only look down. My looking is a little more forward because you look at me. When I want to say something, I say it with a shout! But my truest feelings can never come out. My words are a little less empty because you listen to me. When something is above me, I reach for the stars. But when I feel small, I don't get very far. My standing is a little bit taller because you sit with me. I believe in myself with all of my heart. But what do I do when it's torn all apart? My faith is a little bit stronger because you trusted me. My pen always puts my feelings to the test. I'm not a good writer, but my best is my best. My poems are a little bit dearer because you think of me. Because you, because you, because you.",
+      "source": "Doki Doki Literature Club!",
+      "length": 792,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34963]:
```suggestion
      "length": 788,
```

File: frontend/static/quotes/english.json [hunk 5]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 133,
+      "id": 5929
+    },
+    {
+      "text": "You know what I heard about Amy? Amy likes spiders. Icky, wriggly, hairy, ugly spiders! That's why I'm not friends with her. Amy has a cute singing voice. I heard her singing my favorite love song. Every time she sang the chorus, my heart would pound to the rhythm of the words. But she likes spiders. That's why I'm not friends with her. One time, I hurt my leg really bad. Amy helped me up and took me to the nurse. I tried not to let her touch me. She likes spiders, so her hands are probably gross. That's why I'm not friends with her. Amy has a lot of friends. I always see her talking to people. She probably talks about spiders. What if her friends start to like spiders too? That's why I'm not friends with her. It doesn't matter if she has other hobbies. It doesn't matter if she keeps it private. It doesn't matter if it doesn't hurt anyone. It's gross. She's gross. The world is better off without spider lovers. And I'm gonna tell everyone.",
+      "source": "Doki Doki Literature Club!",
+      "length": 958,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34957]:
```suggestion
      "length": 952,
```

File: frontend/static/quotes/english.json [hunk 6]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 1582,
+      "id": 5928
+    },
+    {
+      "text": "Monkeys can climb. Crickets can leap. Horses can race. Owls can seek. Cheetahs can run. Eagles can fly. People can try. But that's about it.",
+      "source": "Doki Doki Literature Club!",
+      "length": 133,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34951]:
```suggestion
      "length": 140,
```

File: frontend/static/quotes/english.json [hunk 7]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 1582,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34945]:
```suggestion
      "length": 1578,
```

File: frontend/static/quotes/english.json [hunk 8]
```diff
@@ -34902,6 +34902,96 @@
...
+    {
+      "text": "Pen in hand, I find my strength. The courage endowed upon me by my one and only love. Together, let us dismantle this crumbling world. And write a novel of our own fantasies. With a flick of her pen, the lost finds her way. In a world of infinite choices, behold this special day. After all, not all good times must come to an end.",
+      "source": "Doki Doki Literature Club!",
+      "length": 331,
+      "id": 5926
+    },
+    {
+      "text": "The way you glow through my blinds in the morning. It makes me feel like you missed me. Kissing my forehead to help me out of bed. Making me rub the sleepy from my eyes. Are you asking me to come out and play? Are you trusting me to wish away a rainy day? I look above. The sky is blue. It's a secret, but I trust you too. If it wasn't for you, I could sleep forever. But I'm not mad. I want breakfast.",
+      "source": "Doki Doki Literature Club!",
+      "length": 404,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34939]:
```suggestion
      "length": 402,
```

File: frontend/static/quotes/english.json [hunk 9]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 1140,
+      "id": 5924
+    },
+    {
+      "text": "But he wasn't looking at me. Confused, I frantically glance at my surroundings. But my burned eyes can no longer see color. Are there others in this room? Are they talking? Or are they simply poems on flat sheets of paper, the sound of frantic scrawling playing tricks on my ears? The room begins to crinkle. Closing in on me. The air I breathe dissipate before it reaches my lungs. I panic. There must be a way out. It's right there. He's right there. Swallowing my fears, I brandish my pen.",
+      "source": "Doki Doki Literature Club!",
+      "length": 439,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34927]:
```suggestion
      "length": 492,
```

File: frontend/static/quotes/english.json [hunk 10]
```diff
@@ -34902,6 +34902,96 @@
...
+      "source": "Doki Doki Literature Club!",
+      "length": 1140,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34921]:
```suggestion
      "length": 1133,
```

File: frontend/static/quotes/english.json [hunk 11]
```diff
@@ -34902,6 +34902,96 @@
...
+    {
+      "text": "It couldn't have been me. SeeIt couldn't have been me. See, the direction the spackle protrudes. A noisy neighbor? An angry boyfriend? I'll never know. I wasn't home. I peer inside for a clue. No! I can't see. I reel, blind, like a film left out in the sun. But it's too late. My retinas. Already scorched with a permanent copy of the meaningless image. It's just a little hole. It wasn't too bright. It was too deep. Stretching forever into everything. A hole of infinite choices. I realize now, that I wasn't looking in. I was looking out. And he, on the other side, was looking in.",
+      "source": "Doki Doki Literature Club!",
+      "length": 584,
+      "id": 5922
+    },
+    {
+      "text": "The colors, they won't stop. Bright, beautiful colors. Flashing, expanding, piercing. Red, green, blue. An endless cacophony of meaningless noise. The noise, it won't stop. Violent, grating waveforms. Squeaking, screeching, piercing. Sine, cosine, tangent. Like playing a chalkboard on a turntable. Like playing a vinyl on a pizza crust. An endless poem of meaningless.\nLoad Me",
+      "source": "Doki Doki Literature Club!",
+      "length": 370,
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:34915]:
```suggestion
      "length": 377,
```

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]
Incorrect quote lengths.

**Pull request closed without merging**